examples/src/main/scala/sqlest/examples/Extractor.scala

```diff
@@ -21,6 +21,25 @@ object ExtractorExamples extends App with DatabaseExample {
     name = FruitTable.name,
     juiciness = FruitTable.juiciness
   )
+
+  // extractors can be used in the declaration of other extractors
+  lazy val smoothyExtractor = extract[Smoothy](
+    description = SmoothyTable.description,
+    fruits = fruitExtractor.asList
+  ).groupBy(SmoothyTable.id)
+
+  // inner joins can be used as follows
+  val smoothies =
+    select
+      .from(
```

Comment:
Ah - also helpful. I'll update the README at the same time (I took this example from there)